### PR TITLE
Update automation for Divine Castigation to remaster and add automation for Castigating Weapon

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/feats/divine-castigation.json [hunk 1]
```diff
@@ -30,16 +30,61 @@
...
+                        ]
+                    }
                 ],
-                "selectors": [
-                    "damage-received"
+                "property": "traits",
+                "value": "unholy"
+            },
+            {
+                "key": "DamageDice",
```
Comment (simonward) [inline packs/feats/divine-castigation.json:74]: Not sure if there's a better way to automate this, given that it only become spirit and damages creatures with the opposing trait.

File: packs/feat-effects/effect-castigating-weapon.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            "title": "Pathfinder Player Core"
+        },
+        "rules": [
+            {
+                "key": "AdjustStrike",
+                "mode": "add",
+                "predicate": [
+                    "self:trait:holy"
+                ],
+                "property": "traits",
```
Comment (stwlam) [inline packs/feat-effects/effect-castigating-weapon.json:30]:
```suggestion
                "property": "weapon-traits",
```

File: packs/feats/divine-castigation.json (hunk 1 above)
Comment (stwlam) [reply packs/feats/divine-castigation.json:74]: This automates something other than what Divine Castigation does.

Comment (simonward) [reply packs/feats/divine-castigation.json:74]: I don't believe it's currently possible to modify the damage type on application. I'll remove this RE then?

**Pull request merged**